### PR TITLE
add connection_name

### PR DESCRIPTION
**Pull request by erSitzt**
add a friendly name to the connection in rabbitmq

image

### CODE REVIEW

**Comment by erSitzt** [comment]
@bladesling most libraries for rabbitmq already support this and its very helpful when lots of services are using the same rabbitmq.. would be nice if you could add this feature.

My PR might noch be the best way, but it should be easy to add anyway.
Thanks !

File: lib/jackrabbit.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const jackrabbit = (url, logger, options = {}) => {
 
     options.reconnectionTimeout = options.reconnectionTimeout || +process.env.RABBIT_RECONNECTION_TIMEOUT || 2000;
     options.maxRetries = options.maxRetries || +process.env.RABBIT_RECONNECTION_RETRIES || 20;
+    options.connectionName = options.connectionName || +process.env.RABBIT_CONNECTION_NAME || 'jackrabbit';
```
Comment (Akamaozu) [inline lib/jackrabbit.js:16]: @erSitzt i believe the `+` before `process.env` variables ensures the value is a number type, but that isn't needed in this case.

`process.env.RABBIT_CONNECTION_NAME` should be fine.

**Comment by Akamaozu** [comment]
@erSitzt 

PR seems great to me.

Guessing the maintainers are no longer at @pagerinc, so updating this might be a bit more complicated than we'd hope.

File: lib/jackrabbit.js (hunk 1 above)
Comment (erSitzt) [reply lib/jackrabbit.js:16]: Oh yeah :) thats what you get from copy&paste :)

**Pull request closed without merging**